### DISCUSSION TITLE
关于高能进度条的事

### DISCUSSION THREAD

**Discussion by FCSDFSDVXDFG** [the1812/Bilibili-Evolved#3449] [分享 / Share]
众所周知，B站视频进度条上有排波浪叫高能进度条，有个特性就是看过某一段视频，看过的这段进度条颜色就会从灰色变成蓝色，但我最近发现自己没看过的视频也会有高能进度条变蓝的情况，这是B站的问题还是某种诡异bug

**Comment by Geeyun-JY3** [comment]
还真能复现。

我的复现方法：在当前页面打开另一个视频后，高能进度条似乎能继承上一个视频看过的进度（即对应区域变为主题色）
